dip-0003.md

```diff
@@ -211,8 +211,8 @@ Reorganisations of the chain must also correctly undo changes to the chain-tips
 
 A ProRegTx is invalid if any of these conditions are true:
 
-  1. collateralOutpoint can't be found in UTXO
-  2. collateralOutpoint amount isn't equal 1000 DASH
+  1. collateralOutpoint `hash` is empty but no output with 1000 DASH can be found on position `n` in outputs of ProRegTx
```

Comment:
"empty" might not be 100% correct here. It's still there, but "all-zero" or "null"